audio/dlc_oxdoorlock/oxdoorlock.awc.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AudioWaveContainer>
+ <Version value="1" />
+ <ChunkIndices value="True" />
+ <Streams>
+  <Item>
+   <Name>button_remote</Name>
+   <FileName>button_remote.wav</FileName>
+   <Chunks>
+    <Item>
+     <Type>peak</Type>
+    </Item>
+    <Item>
+     <Type>data</Type>
+    </Item>
+    <Item>
+     <Type>format</Type>
+     <Codec>PCM</Codec>
```

Comment:
`ADPCM` should be used over `PCM` for its smaller sizes with no noticeable lost in quality

Reply:
we only figured this out today you rat, but you right I will update it 🤪

Reply:
> we only figured this out today you rat, but you right I will update it 🤪

Stop with the excuses and just do what you are told you minion